docs/cloud/aws/lca1/customer-ssp/index.md

```diff
@@ -0,0 +1,81 @@
+# System Security Plan (SSP) Information
+
+Systems that fall under the [CU Data Classification](https://www.cu.edu/data-governance/resources-support/data-classification) of **Confidential Information** or
+**Highly Confidential Information** are required to provide a System Security Plan (SSP).
+An SSP is a document that outlines how an organization protects the security of a system.
+An SSP is a critical document that helps organizations meet compliance requirements.
+
+The [OIT IT Security](https://oit.colorado.edu/services/it-security) group is responsible for guiding, evaluating and approving your SSP.
```

Comment:
OIT-Sec guides and evaluates.  The PO is responsible for the approval it sounds like.  This is muddy territory.

Reply:
Removed 'approval'